src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -1,7 +0,0 @@
-// Elastic charts
-@import '@elastic/charts/dist/theme';
-@import '@elastic/eui/src/themes/charts/theme';
```

Comment:
Did these imports get moved somewhere?

Reply:
After some investigation: these imports were dead for a long time. The actual imports of the EUI themes are done here: 

https://github.com/elastic/kibana/blob/481dccdb6f20daf3bbb65d239dca5a1f5a579d9c/packages/kbn-ui-shared-deps/webpack.config.js#L35-L42

Reply:
I don't know if you're waiting on my review, but are you going to/did you create an issue to change these imports back to SASS?

Reply:
Created https://github.com/elastic/kibana/issues/76475. Wasn't sure which (other) teams to include on it, feel free to add any if relevant.

Reply:
Thanks!